src/ui/profile_preferences.ui

```diff
@@ -1991,11 +2015,11 @@ you can use it but there could be issues with aligning columns of text</string>
              <string>&lt;p&gt;Select profiles that you want to copy map to, then press the Copy button to the right&lt;/p&gt;</string>
             </property>
             <property name="text">
-             <string>Press to pick destination(s)</string>
+             <string comment="text on button to select other profiles to receive the map from this profile, this is used when no profiles have been selected">pick destinations ...</string>
```

Comment:
```suggestion
             <string comment="text on button to select other profiles to receive the map from this profile, this is used when no profiles have been selected">pick destinations...</string>
```

Reply:
I've just been reading up on this - and it seems that your proposal is (one) American style - and that it varies across locales... https://en.wikipedia.org/wiki/Ellipsis#In_different_languages - of note is https://en.wikipedia.org/wiki/Ellipsis#Usage_in_computer_system_menus - which does suggest it is appropriate to use them as we have here to suggest that a further dialogue will happen on activation/clicking rather than an operation being carried out directly. :grin:

Reply:
Actually we've been there before. For example only some of the actions in the mapper's context menu end in ellipsis. That is deliberate. 😉